piccolo/apps/migrations/auto/migration_manager.py

```diff
@@ -713,6 +747,13 @@ async def _run_rename_tables(self, backwards: bool = False):
                 },
             )
 
+            await self._run_query(
+                _Table.alter().rename_pk_constraint(
+                    old_constraint_name=f"{tablename}_pkey",
+                    new_constraint_name=f"{new_tablename}_pkey",
+                )
+            )
```

Comment:
I think we should leave this out for now. Because renaming tables currently works with SQLite, but renaming constraints doesn't.

Reply:
We could leave this and skip the renaming constraint for `Sqlite`. Something like this
```python
engine = engine_finder()

if engine.engine_type in ("postgres", "cockroach"):
    await self._run_query(
        _Table.alter().rename_constraint(
            old_constraint_name=f"{tablename}_pkey",
            new_constraint_name=f"{new_tablename}_pkey",
        )
    )

await self._run_query(
    _Table.alter().rename_table(new_name=new_tablename)
)
```

Reply:
Yeah, that works, but I'm tempted to just leave out the renaming of the primary key constraint though - we don't need it for the `on delete` / `on update` changes. This is already a pretty big PR, and I'm always nervous about changing too much stuff with migrations.

Reply:
Sorry. I will revert the latest commit.